### DISCUSSION TITLE
Negative optimism for net benefit

### DISCUSSION THREAD

**Discussion by Meech81** [MSKCC-Epi-Bio/decisioncurveanalysis#18] [Q&A] [answered]
After carrying out 200 bootstraps to estimate the optimism around the net benefit of a model developed using stepwise BIC logistic regression, I'm getting a negative value meaning that when subtracted from the apparent net benefit I end up with an improvement. Has anyone else come across this before? I'm not looking for advice on variable selection methods, I realise there are alternatives to stepwise which are more robust.

**Comment by VickersA** [comment]
Bootstrap corrected net benefit via optimism is not recommended for net benefit. We recommend crossvalidation instead see https://pubmed.ncbi.nlm.nih.gov/19036144/. We suggest doing the crossvalidation and then using the predicted probabilities thereby obtained for discrimination, calibration and net benefit.

**Reply by Meech81** [reply]
Thank you, should I expect not to see negative optimism if I use cross validation in place of bootstrapping? And if I do still get negative optimism, what does that mean?

**Reply by VickersA** [reply]
Optimism is not something you calculate during crossvalidation, it has no meaning in that context.

**Comment by Meech81** [comment]
Oh yes, I see. I guess the question in the cross validation context would be can corrected net benefit be higher than uncorrected net benefit?

**Comment by VickersA** [comment, marked as answer]
Net benefit (NB) and corrected net benefit (NBc) are estimates associated with error. True NB> True NBc but if True NB - True NBc is small, it is possible that estimated NB < estimates NBc
